utility_network/perform-valve-isolation-trace/README.md

```diff
@@ -0,0 +1,53 @@
+# Perform valve isolation trace
+
+Run a filtered trace to locate operable features that will isolate an area from the flow of network resources.
+
+![Image of a utility network with an isolation trace applied to it](PerformValveIsolationTrace.png)
+
+## Use case
+
+Determine the set of operable features required to stop a network's resource, effectively isolating an area of the network. For example, you can choose to return only accessible and operable valves: ones that are not paved over or rusted shut.
+
+## How to use the sample
+
+Create and set the configuration's filter barriers by selecting a category. Check or uncheck 'Include Isolated Features'. Click 'Trace' to run a subnetwork-based isolation trace.
```

Comment:
just adding a suggestion for the updated UI text case I've suggested later
```suggestion
Create and set the configuration's filter barriers by selecting a category. Check or uncheck 'Include isolated features'. Click 'Trace' to run a subnetwork-based isolation trace.
```

Reply:
Thanks, I've also updated to use `"` instead of `'`